### PR TITLE
add python-google api bionic, focal support

### PR DESCRIPTION
**Pull request by mqcmd196**
Please add the following dependency to the rosdep database.

## Package name:

python-googleapi

## Purpose of using this:

I want to install this package with ubuntu ros melodic, noetic environment

### CODE REVIEW

File: rosdep/python.yaml [hunk 1]
```diff
@@ -1988,6 +1988,8 @@ python-googleapi:
   fedora: [google-api-python-client]
   gentoo: [dev-python/google-api-python-client]
   ubuntu:
+    bionic: [python-googleapi]
+    focal: [python3-googleapi]
```
Comment (tfoote) [inline rosdep/python.yaml:1992]: Following our conventions this key is for Python 2 /any. We should not mix in the explicit python3 version here. It should get a new key `python3-googleapi`

**Review by tfoote — CHANGES_REQUESTED** [review]
I see the python2 version tops out at bionic: https://packages.ubuntu.com/bionic/python-googleapi

And the python3 version starts at least at bionic: https://packages.ubuntu.com/impish/python3-googleapi

File: rosdep/python.yaml (hunk 1 above)
Comment (mqcmd196) [reply rosdep/python.yaml:1992]: Thanks, I'll fix

Comment (mqcmd196) [reply rosdep/python.yaml]: fixed

File: rosdep/python.yaml [hunk 2]
```diff
@@ -1988,6 +1988,9 @@ python-googleapi:
   fedora: [google-api-python-client]
   gentoo: [dev-python/google-api-python-client]
   ubuntu:
+    bionic: [python-googleapi]
+    bionic_python3: [python3-googleapi]
+    focal_python3: [python3-googleapi]
```
Comment (clalancette) [inline rosdep/python.yaml:1993]: This style is deprecated, and we don't use it anymore.  Please make an entirely new rosdep key called `python3-googleapi`, and populate it with these entries.

**Review by clalancette — CHANGES_REQUESTED** [review]

File: rosdep/python.yaml (hunk 2 above)
Comment (jacobperron) [reply rosdep/python.yaml:1993]: @mqcmd196 Friendly ping. Please move these Python3 packages to a new rosdep key as Chris suggests.

Comment (mqcmd196) [reply rosdep/python.yaml]: sorry,,, I fixed.

**Review by clalancette — CHANGES_REQUESTED** [review]
It looks like this package is also available for Debian under the same name, so please add a section for Debian as well.

File: rosdep/python.yaml [hunk 3]
```diff
@@ -6551,6 +6552,11 @@ python3-gnupg:
   ubuntu: [python3-gnupg]
 python3-google-cloud-storage-pip: *migrate_eol_2025_04_30_python3_google_cloud_storage_pip
 python3-google-cloud-texttospeech-pip: *migrate_eol_2025_04_30_python3_google_cloud_texttospeech_pip
+python3-googleapi:
+  debian: [python3-googleapi]
+  ubuntu:
+    bionic: [python3-googleapi]
+    focal: [python3-googleapi]
```
Comment (clalancette) [inline rosdep/python.yaml:6559]: Sorry, I missed this on the last review.  There is no need to split this up, so just make this:

```suggestion
  ubuntu: [python3-googleapi]
```

**Review by clalancette — CHANGES_REQUESTED** [review]

File: rosdep/python.yaml (hunk 3 above)
Comment (mqcmd196) [reply rosdep/python.yaml]: Thank you for your kind review, fixed

**Review by clalancette — APPROVED** [review]
Thanks for iterating, looks good to me.

**Review by ivanpauno — APPROVED** [review]

**Pull request merged**